packages/webostv/template/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my-app",
+  "version": "1.0.0",
+  "description": "A general template for an Enact Moonstone application for webOS TVs",
+  "author": "",
+  "main": "src/index.js",
+  "scripts": {
+    "serve": "enact serve",
+    "pack": "enact pack",
+    "pack-p": "enact pack -p",
+    "watch": "enact pack --watch",
+    "clean": "enact clean",
+    "lint": "enact lint .",
+    "license": "enact license",
+    "test": "enact test start --single-run",
+    "test-watch": "enact test start"
+  },
+  "license": "UNLICENSED",
+  "private": true,
+  "repository": "",
+  "enact": {
+    "theme": "moonstone"
+  },
+  "eslintConfig": {
+    "extends": "enact"
+  },
+  "eslintIgnore": [
+    "node_modules/*",
+    "build/*",
+    "dist/*"
+  ],
+  "dependencies": {
+    "@enact/core": ">=2.0.0-alpha <2.0.0",
```

Comment:
I'm curious about this range.  This would prevent apps based on this template from updating to `2.0.0` when released, would it not?  Why would we want to prevent that?

Reply:
Correct. All templates on `develop` use this range (and publishes under NPM `next` tag) to always get the in-develop release. As far as I'm aware this template is being added to `develop` and has no need for a current `master` 1.x stable release range.  All templates will be updates for 2.x stables together when the time comes.